mediapipe/tasks/ios/test/vision/utils/sources/parse_proto_utils.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2022 The MediaPipe Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "mediapipe/tasks/ios/test/vision/utils/sources/parse_proto_utils.h"
+
+#include <fstream>
+#include <sstream>
+
+namespace mediapipe::tasks::ios::test::vision::utils {
+
+namespace {
+  using ::google::protobuf::TextFormat;
+ }
+
+ absl::Status get_proto_from_pbtxt(const std::string file_path, google::protobuf::Message& proto) {
+    
+    std::ifstream file_input_stream(file_path);
+    if(!file_input_stream.is_open()) return absl::InvalidArgumentError(
+          "Cannot read input file.");
+    
+    std::stringstream strings_stream ;
+    strings_stream << file_input_stream.rdbuf();
+
+    return TextFormat::ParseFromString(strings_stream.str(), &proto) ? absl::OkStatus() : absl::InvalidArgumentError(
```

Comment:
@schmidt-sebastian I think the [C++ tests](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/cc/vision/pose_landmarker/pose_landmarker_test.cc#L77) uses `GetTextProto` from some third party library for reading a proto from a file.  I can't find the header which imports the declaration of `GetTextProto`. It looks like a method from [google or tools](https://google.github.io/or-tools/cpp/namespacefile.html#a5cb1dcdd93aee8846abedf0f5905ba65). 
It seems to be a transitive dependency of one of the header files imported in the C++ tests.
We don't explicitly depend on `or tools` from the work space. It may be installed as part of some other library. Since C++ tests are not enable, I am not too sure about this as I couldn't run the tests. Hence I ended up implementing this separately for the iOS landmark tests. If you can find out where `GetTextProto` is defined, I will remove this util and use that instead.

Reply:
I actually tried to get this to work for the FaceDetector tests but ultimately gave up. You cannot use GetTextProto in open-source code. There is no BUILD file for these tests - https://github.com/google/mediapipe/blob/master/mediapipe/tasks/cc/vision/pose_landmarker/BUILD - so they are not compiled. I got confused by this as well.

Thank you for coming up with a solution.